api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -183,6 +183,20 @@ message OktaResourcesUpdatedMetadata {
   int32 Updated = 2 [(gogoproto.jsontag) = "updated"];
   // Deleted is the number of resources deleted.
   int32 Deleted = 3 [(gogoproto.jsontag) = "deleted"];
+  // AddedResources is a list of the actual resources that were added.
+  repeated OktaResource AddedResources = 4 [(gogoproto.jsontag) = "added_resources"];
+  // AddedResources is a list of the actual resources that were updated.
```

Comment:
Should this not be "UpdatedResources is a list of the actual resources that were updated."?

Reply:
Ahh whoops, bad copy paste!